_data/simple-icons.json

```diff
@@ -3521,4 +3526,4 @@
             "source": "https://github.com/zulip/zulip/"
         }
     ]
-}
+}
```

Comment:
Small request, but could you revert this change @mvedataydin? It's not a big deal, especially not in this project, but I think it is good practice not to pollute `git blame` and `git diff`. Especially in bigger/more complex projects and/or larger Pull Requests :slightly_smiling_face: 

You might have to do something with the configuration of your editor/IDE. Alternatively you can use a simple text editor (like notepad.exe or gedit) or even the GitHub online IDE to fix this. Let me know if you can't figure this out.

Reply:
Thank you a lot for the feedback,I am slightly new to contributing to open source. I was trying to figure out and  solve the issue and mistakenly closed the pr. Do you have any suggestions to me with best practices of contributing? Thank you in advance.